README.rst

```diff
@@ -85,6 +85,17 @@ and ``password`` from that file as login credentials.
    { "username": "myusername", "password": "mypassword" }
 
 
+You can also provide multiple username and password tuples in a file as login credentials.
+The first username and password specified will be used as default if pycaching.login() is
```

Comment:
Explaining the default is fine.

In addition, it would be nice to explain the other use case: that you can provide only a username and pycaching then walks throught the whole list and selects the right password.

Reply:
Sure, I will add the intended usage.

Reply:
I added a new sentence covering the topic requested.